src/test/java/fr/esgi/al5_2/Tayarim/ProprietaireControllerTest.java

```diff
@@ -18,297 +16,366 @@
-                .contentType(MediaType.APPLICATION_JSON)
-                .content(objectMapper.writeValueAsString(proprietaireUpdateDTO)));
-
-        response.andExpect(MockMvcResultMatchers.status().isOk())
-                .andExpect(MockMvcResultMatchers.jsonPath("$.nom", CoreMatchers.is(proprietaireDTO.getNom())))
-                .andExpect(MockMvcResultMatchers.jsonPath("$.prenom", CoreMatchers.is(proprietaireUpdateDTO.getPrenom())))
-                .andExpect(MockMvcResultMatchers.jsonPath("$.email", CoreMatchers.is(proprietaireDTO.getEmail())))
-                .andExpect(MockMvcResultMatchers.jsonPath("$.numTel", CoreMatchers.is(proprietaireDTO.getNumTel())))
-                .andExpect(MockMvcResultMatchers.jsonPath("$.logements", CoreMatchers.is(proprietaireDTO.getLogements())));
-    }
-
-    @Test
-    public void ProprietaireController_DeleteProprietaire_ReturnOk() throws Exception {
-        given(proprietaireService.deleteProprietaire(1L)).willAnswer(invocationOnMock -> proprietaireDTO);
-
-        when(authService.verifyToken(any())).thenReturn(proprietaireDTO); //skip token verif
-
-        ResultActions response = mockMvc.perform(delete("/proprietaires/1")
-                .header("Authorization", "Bearer eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJqb2huZG9lQGdtYWlsLmNvbTs2MjZjMjlhNS0zZGU0LTQ5Y2YtODI4ZS1hNDkxNWQzMzM5N2EiLCJpYXQiOjE3MTQxMTM1NzcsImV4cCI6MTcxNDExNzE3N30.kbU1pVkUHkRXktX44JBFN_xzDv-ZvSmtPjnjORO0vHPiHd3f2MGfDF15VTFO5icIrU_bV9cTqZ70RDlKdlp0-g")
-                .contentType(MediaType.APPLICATION_JSON));
-
-        response.andExpect(MockMvcResultMatchers.status().isOk())
-                .andExpect(MockMvcResultMatchers.jsonPath("$.nom", CoreMatchers.is(proprietaireDTO.getNom())))
-                .andExpect(MockMvcResultMatchers.jsonPath("$.prenom", CoreMatchers.is(proprietaireDTO.getPrenom())))
-                .andExpect(MockMvcResultMatchers.jsonPath("$.email", CoreMatchers.is(proprietaireDTO.getEmail())))
-                .andExpect(MockMvcResultMatchers.jsonPath("$.numTel", CoreMatchers.is(proprietaireDTO.getNumTel())))
-                .andExpect(MockMvcResultMatchers.jsonPath("$.logements", CoreMatchers.is(proprietaireDTO.getLogements())));
-    }
+
+  @Autowired
+  private MockMvc mockMvc;
+
+  @MockBean
+  private ProprietaireService proprietaireService;
+
+  @MockBean
+  private AuthService authService;
+
+  @Autowired
+  private ObjectMapper objectMapper;
+  private ProprietaireCreationDto proprietaireCreationDTO;
+  private Proprietaire proprietaire;
+  private List<ProprietaireDto> proprietaireDtos;
+  private ProprietaireDto proprietaireDTO;
+  private ProprietaireDto proprietaireDto2;
+  private ProprietaireDto proprietaireDtoWithLogement;
+  private ProprietaireUpdateDto proprietaireUpdateDTO;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'proprietaireUpdateDTO' must contain no more than '1' consecutive capital letters.